.github/workflows/test-and-build.yml

```diff
@@ -62,9 +62,9 @@ jobs:
         with:
           path: |
             models/
-          key: ${{ runner.os }}-models-v1-${{ hashFiles('tools/make/models.mk') }}
+          key: ${{ runner.os }}-models-v2-${{ hashFiles('tools/make/models.mk') }}
```

Comment:
Why is v2 being used here? Is it really necessary for us to upgrade to v2? The model files are very large and will occupy a lot of cache space on GitHub.